README.md

```diff
@@ -237,7 +237,7 @@ The startup benchmark measures from when the benchmark.js file loads to the call
 
 To use:
 
-*It is recommended that a dedicated Firefox profile is used with the about:config preference of `security.turn_off_all_security_so_that_viruses_can_take_over_this_computer` set to true so garbage collection and cycle collection can be run in between test rounds*
+*It is recommended that a dedicated Firefox profile is used with the about:config preference of `security.turn_off_all_security_so_that_viruses_can_take_over_this_computer` set to true so garbage collection and cycle collection can be run in between test rounds. To do this on a Firefox OS device, see: https://wiki.mozilla.org/B2G/QA/Tips_And_Tricks#For_changing_the_preference*
```

Comment:
There's actually a colon at the end of that URL! I.e. it's:

https://wiki.mozilla.org/B2G/QA/Tips_And_Tricks#For_changing_the_preference:

GitHub won't auto-linkify it correctly, though, so I would make this something like:

To do this on a Firefox OS device, see [B2G/QA/Tips And Tricks](https://wiki.mozilla.org/B2G/QA/Tips_And_Tricks#For_changing_the_preference:).